frontend/src/utils/Wrapper.jsx

```diff
@@ -0,0 +1,15 @@
+import Navbar from "../components/Navbar"
+import Footer from "../components/footer"
+import RoutesComponent from "../routes/RoutesComponent"
+
+const Wrapper = () => {
```

Comment:
I don't know if this Wrapper needs to be extracted into its own component, this adds more complexity. you can just import the Navbar and Footer in the App component

Reply:
Then again the Navbar and Footer are rendered conditionally across some other screens of the whole web app, so you might want to prepare for that now